samplesheet_bc_dx.csv

```diff
@@ -1,4 +1,4 @@
-sample,input_file,barcode,genome,transcriptome,condition
-K562_RUN1_REP3,,1,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,,
-HEPG2_RUN3_REP5,,2,,,
-CALIBRATION_RUN,,3,,,
+group,replicate,input_file,barcode,genome,transcriptome
+,3,K562_RUN1_REP3,1,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,
```

Comment:
```suggestion
K562_RUN1_REP3,1,,1,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,
```

Reply:
The rows don't look right for these files? We don't need `REP3` in the name of the file because we can add that in within the pipeline using the `replicate` column.

Reply:
On second thoughts I think `group,replicate,barcode,input_file,genome,transcriptome` would be better i.e. switching the `input_file` and `barcode` columns.

Reply:
Thanks for the comments, @drpatelh! I have modified the samplesheets and check_samplesheet.py accordingly. The tests ran through locally. [Here](https://github.com/GoekeLab/nanoseq/pull/10#issuecomment-676050100) are all the runs with the modified samplesheets.